IS7024/Pages/Index.cshtml

```diff
@@ -8,7 +8,7 @@
     <div class="text-center">
         <h1 class="display-4">Welcome</h1>
         <h4 class="display-4">Please Select a Concert Venue Below</h4>
-        </hr>
```

Comment:
Took off the end tag part of this tag.

Reply:
It belongs in the tag, it just belongs on the other side.
In other words, should be hr/, not /hr.